### DISCUSSION TITLE
xds.b Listener for ITI-41 Channel

### DISCUSSION THREAD

**Discussion by rharris2017** [nextgenhealthcare/connect#5637] [Q&A]
I am new to Mirth and learning.
We purchased the [SW LIC,Connect Platinum-MNTNC,CNCTD HLTH
Running Mirth ver 3.12
I have the channel setup and am able to receive. 

However, I have been asked to modify the response. 

I did not build the response they received and don't a response in the channel that I can edit.

I am, not find any details on using the interoperability suite other than the user guide that covers it at 20000 feet.
Any guidance I can get would be great.

**Comment by jonbartels** [comment]
> Connect Platinum

You're paying at least $30.000/year for that. Ask for a login for your company on the NextGen Success Community and open a ticket.

> I did not build the response they received and don't a response in the channel that I can edit.

I'm not sure what you mean. Can you show some screenshots for where you're looking?

**Comment by rharris2017** [comment]
I did open a ticket and they sent me this 19-page Interoperability Connector Suite User Guide (3.6)[]
[Interoperability Connector Suite User Guide (3.6).pdf](https://github.com/nextgenhealthcare/connect/files/10511562/Interoperability.Connector.Suite.User.Guide.3.6.pdf)

I was set to Auto Generate.

**Reply by pacmano1** [reply]
Much like any other connector you can build the response yourself.
